test/domassist.test.js

```diff
@@ -50,6 +50,20 @@ test('addClass, hasClass, removeClass', assert => {
   assert.end();
 });
 
+test('toggleClass', assert => {
```

Comment:
I'd just put this all into this test case and use the existing hasClass: https://github.com/firstandthird/domassist/blob/master/test/domassist.test.js#L42

Also, don't need to do any teardown for these.